sdk/communication/azure-communication-network-traversal/azure/communication/network/traversal/_shared/models.py

```diff
@@ -0,0 +1,5 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+# pylint: skip-file
```

Comment:
The shared folder should be consistent across communication python SDKs, since its the equivalent of the common package in the other languages.